From 1c67184578c5d1ee17721d709f9aea6242c90904 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Thu, 4 Apr 2019 10:03:35 -0700
Subject: [PATCH 0618/1851] uio: xilinx_ai_engine: Enable platform genirq
 driver

commit 79c3b44d7980b435f77cf436f96c1e828d36c954 from
https://github.com/Xilinx/linux-xlnx.git

The NPI can be modeled with the platform genirq driver. Enable
the driver as a part of it. This will require the dts node
for the NPI space.

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Reviewed-by: Wendy Liang <wendy.liang@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/uio/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/uio/Kconfig b/drivers/uio/Kconfig
index 799891b52688..b3b157324d21 100644
--- a/drivers/uio/Kconfig
+++ b/drivers/uio/Kconfig
@@ -171,6 +171,7 @@ config UIO_XILINX_AI_ENGINE
 	tristate "Xilinx AI Engine driver"
 	select IRQ_SIM
 	select UIO_DMEM_GENIRQ
+	select UIO_PDRV_GENIRQ
 	help
 	  The driver for Xilinx AI Engine that utilizes the uio_dmem_genirq.
 	  The userspace library will use this to interact with the AI Engine
-- 
2.31.1

